Cargo.toml

```diff
@@ -8,4 +8,5 @@ authors = ["Polytech Montpellier - DevOps"]
 clap = { version = "3.0.5", features = ["derive"] }
 lazy_static = "1.4.0"
 oci-spec = "0.5.3"
-unshare = { git = "https://github.com/virt-do/unshare", branch = "main" }
+# unshare = { git = "https://github.com/virt-do/unshare", branch = "main" }
+unshare = { path = "../unshare" }
```

Comment:
If you fixed/improved the `unshare`crate, please send a PR against https://github.com/virt-do/unshare